### ISSUE TITLE
History of adrenalectomy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10289]
**Preferred term label:**
History of adrenalectomy

**Synonyms**
surgical removal of the adrenal glands

**Definition (free text, please give PubMed ID)**
surgical removal of the adrenal glands

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Post-adrenalectomy syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000194
- label: History of adrenalectomy
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:28421470
- def: A past medical history of havind had a surgical procedure to remove the adrenal glands.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10289
- synonyms:
